Myob.Fma.BookingLibrary/BorrowedItemsManagement/BorrowingManager.cs

```diff
@@ -7,29 +7,23 @@
 using Myob.Fma.BookingLibrary.Memberships;
 using Myob.Fma.BookingLibrary.Resources;
 
-namespace Myob.Fma.BookingLibrary.BorrowedItemsManagment
+namespace Myob.Fma.BookingLibrary.BorrowedItemsManagement
 {
     public class BorrowingManager : IBorrowingManager
     {
-        private readonly List<IBorrowedItem> _borrowedItems = new List<IBorrowedItem>();
+        private readonly List<IBorrowedItem> _borrowedItemsHistory = new List<IBorrowedItem>();
 
-        public void BorrowItem(IResource resource, IMembership membership)
+        public void AddEntryToBorrowedItemsHistory(IResource resource, IMembership membership)
         {
-            var borrowedItem = new BorrowedItem()
-            {
-                Resource = resource,
-                Member = membership,
-                DueDate = membership.GetDateMemberCanBorrowTill(),
-                BorrowedDate = DateTime.UtcNow,
-            };
+            var borrowedItem = CreateBorrowedItem(resource, membership);
 
-            _borrowedItems.Add(borrowedItem);
+            _borrowedItemsHistory.Add(borrowedItem);
         }
 
         public void ReturnItem(IResource resource, IMembership membership)
         {
             var returnedItem =
-                _borrowedItems.First(i =>
+                _borrowedItemsHistory.First(i =>
```

Comment:
I googled it and I found the following: 
```
`FirstOrDefault()` is just like `First()` except that, if no element match the specified 
condition than it returns default value of underlying type of generic collection. 
It does not throw `InvalidOperationException` if no element found. 
```
In this case you want it to return null if it's not exist and not throw an exception.